README.md

```diff
@@ -40,7 +40,11 @@ pip install pyElli[fitting]
 ```
 
 This installs pyElli with the additional fitting capabilities and interactive widgets.
-If don't want to have this functionality just drop the `[fitting]` in the end.
+If you don't want to have this functionality just drop the `[fitting]` in the end.
+
+To increase performance of the 4x4 Solver by a factor of 20, it is recommended to
+install PyTorch CPU package manually, as it is too heavy to include in the standard installation.
```

Comment:
We don't necessarily need the CPU version, right?

Reply:
No not strictly, the GPU versions are fine too, but way bigger, and we would not get any benefits from using them.